lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -438,6 +456,109 @@ def restart_services(self, services):
         """
         self._magmad_client.restart_services(services)
 
+    def create_mme_config_backup(self):
+        """
+        This function creates a backup of default MME configuration file,
+        which can later be used to restore the original configuration
+        """
+
+        print("Creating a backup of MME configuration file")
+        self.exec_command(
+            "sudo cp -n "
+            + self.mme_config_file
+            + " "
+            + self.mme_config_file
+            + "_bak"
```

Comment:
```suggestion
            + ".bak"
```

Reply:
Changed as per the suggestion and updated the PR